sermant-plugins/sermant-router/spring-router-plugin/src/main/java/io/sermant/router/spring/interceptor/EurekaHttpClientInterceptor.java

```diff
@@ -53,14 +53,14 @@ public ExecuteContext before(ExecuteContext context) {
         if (argument instanceof InstanceInfo) {
             InstanceInfo instanceInfo = (InstanceInfo) argument;
             AppCache.INSTANCE.setAppName(instanceInfo.getAppName());
+            configService.init(RouterConstant.SPRING_CACHE_NAME, AppCache.INSTANCE.getAppName());
```

Comment:
String serviceName = instanceInfo.getAppName()
AppCache.INSTANCE.setAppName(serviceName);
configService.init(RouterConstant.SPRING_CACHE_NAME, serviceName);

Reply:
in eureka, appname is a stirng